tests/helpers.py

```diff
@@ -147,6 +147,16 @@ def assert_polygon_annotation_matches_dict(
         assert instance_pt["y"] == dict_pt["y"]
 
 
+def assert_segmentation_annotation_matches_dict(
+    annotation_instance, annotation_dict
+):
+    assert annotation_instance.mask_url == annotation_dict["mask_url"]
+    # Cannot guarantee segments are in same order
```

Comment:
Could consider making segments comparable and then sorting them, and checking the sorted lists are equal

Reply:
done, great suggestion!